intro-checkpoint.ipynv

```diff
@@ -0,0 +1,172 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "74c98243",
+   "metadata": {},
+   "source": [
+    "# Write a program to create \"intro.txt\" file.  Print only even number of row."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47c567ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "id": "c96c2ef4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=open('intro.txt','w',newline=\"\")\n"
```

Comment:
rewrite this statement other way?

Reply:
**with open('intro.txt','w',newline="")as k:**